subprojects/parseq-restli-client/src/main/java/com/linkedin/restli/client/DirectExecutor.java

```diff
@@ -0,0 +1,21 @@
+package com.linkedin.restli.client;
+
+import java.util.concurrent.Executor;
+
+
+/**
+ * A simple executor implementation that executes the task immediately on the calling thread
+ */
+class DirectExecutor implements Executor {
```

Comment:
is this similar to MoreDirectors.directDirector() ?

Reply:
Yes. but @zackthehuman prefers not introducing dependencies on Guava.

Reply:
Correct; I don't see any other usage of Guava in the `parseq-restli-client` module, and I think it's a bit heavy-handed to introduce it just so we can use the `DirectExecutor`. I see two other options here:

1) Implement a simple `DirectExecutor` (which is what you've done)
2) Make the `_executor` nullable and then if it's `null` just don't use an executor to run `sendRequest` (which is what the code did prior to this PR)